README.md

```diff
@@ -21,7 +21,7 @@ docker build --build-arg PACKS="<pack names>" -t ${DOCKER_REGISTRY}/st2packs:<ve
 ```
 
 where `<pack names>` is a space separated list of packs you want to install in the st2packs image
-and `<docker_registry>` is the registry URL. '<version number>' should be used and incremented with each change to the pack contents, since using 'latest' as the pack will not tell helm when changes occur and upgrade will not occur. The pack may be specified as a github url, or as a
+and `<docker_registry>` is the registry URL. `<version number>` should be used and incremented with each change to the pack contents, since using 'latest' as the pack will not tell helm when changes occur and upgrade will not occur. The pack may be specified as a github url, or as a
```

Comment:
I think it would be even better using `<version_number>` for consistency. Same in: 

```diff
- docker build --build-arg PACKS="<pack names>" -t ${DOCKER_REGISTRY}/st2packs:<version number> st2packs-image
+ docker build --build-arg PACKS="<pack names>" -t ${DOCKER_REGISTRY}/st2packs:<version_number> st2packs-image
```

Reply:
Ah, agreed, consistency would be good.